source/dub/generators/generator.d

```diff
@@ -259,6 +259,20 @@ class ProjectGenerator
 
 		auto roottarget = &targets[rootPackage.name];
 
+		// Handle the destination directory being overwritten.
```

Comment:
```suggestion
		// Handle the destination directory being overridden.
```
Slight confusion I had when first looking at this. I thought you meant that the output directory was going to be overwritten with new files.